static/docs/commands-reference/version.md

```diff
@@ -20,6 +20,24 @@ system/environment:
 | `Python version` | Version of the Python being used for the project in which DVC is initialized   |
 | `Platform`       | Information about the operating system of the machine                          |
 
+#### Components of DVC version
+
+The detail of DVC version depends upon the way of installing the project.
+
+- **`pip install git+git://github.com/iterative/dvc`**
+  - This will install dvc using the `master` branch of DVC's repository. The
+    master branch is continuously being updated with changes which might not be
+    ready to publish yet. Therefore installing using the above command might
+    have issues regarding its usage. So to trace any error reported with this
+    setup, we need to know exactly which version is being used. For this, we
+    rely on git commit hash which is displayed in output as
+    `BASE_VERSION+COMMIT_HASH`.
+- **`pip install dvc`**
+  - This installs dvc using the official release stored in Python Packaging
```

Comment:
why do we need a second level list here?

Reply:
It was making it look clean. What do you suggest?
Is this fine?
![Screenshot from 2019-05-20 22-27-17](https://user-images.githubusercontent.com/35191225/58038752-7ad5ac80-7b4e-11e9-81f9-6a4bc4b50e85.png)

Reply:
Or instead of `:`, we can use &rarr;.